.github/actions/checkout-submodules-and-bootstrap/action.yaml

```diff
@@ -26,6 +30,9 @@ runs:
       uses: ./.github/actions/bootstrap
       with:
         platform: ${{ inputs.platform }}
+    - name: Dump disk info after checkout submodule & Bootstrap
```

Comment:
should we use one consistent way both here and above?

I believe the running of a bash shell script is the more straight forward one without a sub-action.

Reply:
Maybe add a comment explaining why we do this on top of the action (no need for post here, but still want to see sizes).